docs/core/settings/variables.md

```diff
@@ -356,3 +356,45 @@ Examples:
 # If %{user} is "testuser", return "INVALID". Otherwise return %{user} uppercased.
 %{user | if ("=", "testuser, "invalid", user) | upper }
 ```
+
+## Cryptography support
+
+### Parameters
+
+| Key                | Value                                                                                                       |
+| ------------------ | ----------------------------------------------------------------------------------------------------------- |
+| key                | Encryption key, if no salt is given, must be hex encoded and match the expected length of chosen algorithm. |
+| iv                 | Initialization vector, provide if no salt is given and algorithm requires one. Must be hex encoded and match the expected length of chosen algorithm. |
+| salt               | If provided, will use hash algorithm to create cipher key and IV with PBKDF2 algorithm. |
+| hash               | Hash to use in PBKDF2. Defaults to SHA-256. |
+| rounds             | Number of rounds to use in PBKDF2. Defaults to 1000. |
+| algorithm          | Encryption algorithm. Expects OpenSSL naming. Some algorithms are not usable due to system or functional restrictions. |
+| raw                | When set to 1, will return encrypted result in binary. |
```

Comment:
How about: `0 = Use structured output format, 1 = Use raw output format.` with links to below chapters.